packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx

```diff
@@ -73,6 +73,32 @@ const styles = (theme: ThemeType) => ({
   podcastsSection: {
     rowGap: '6px',
   },
+  wrappedAd: {
+    backgroundColor: theme.palette.wrapped.background,
+    color: theme.palette.text.alwaysWhite,
+    padding: '12px 24px',
+    borderRadius: theme.borderRadius.default,
+    '&:hover': {
+      opacity: 1,
+      backgroundColor: `color-mix(in oklab, ${theme.palette.wrapped.background} 85%, ${theme.palette.text.alwaysBlack})`,
+      color: `color-mix(in oklab, ${theme.palette.text.alwaysWhite} 90%, ${theme.palette.text.alwaysBlack})`,
```

Comment:
I guess this is going to change based on Agnes' comment, but NB for the future, [browser support for color-mix](https://caniuse.com/?search=color-mix) is pretty bad ATM (I usually try to keep to at least 95%, and it's only 85%)

Reply:
Personally I think that's fine, it has support in all major browsers, [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix) doesn't even list the ones it doesn't support